lib/get-prefix.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+// This attempts to find the longest common prefix of strings in an array
```

Comment:
Does this function just help readability?  What do the strings look like that it's going to be stripping later?

Reply:
This switches the report to use relative paths instead of absolute paths. If tap is run inside a container, it will look like this:

```
{"total": {"lines":{"total":228,"covered":228,"skipped":0,"pct":100},"statements":{"total":239,"covered":239,"skipped":0,"pct":100},"functions":{"total":94,"covered":94,"skipped":0,"pct":100},"branches":{"total":87,"covered":87,"skipped":0,"pct":100}}
,"/opt/app/lib/bool.js": {"lines":{"total":26,"covered":26,"skipped":0,"pct":100},"functions":{"total":6,"covered":6,"skipped":0,"pct":100},"statements":{"total":27,"covered":27,"skipped":0,"pct":100},"branches":{"total":13,"covered":13,"skipped":0,"pct":100}}
,"/opt/app/lib/constants.js": {"lines":{"total":1,"covered":1,"skipped":0,"pct":100},"functions":{"total":0,"covered":0,"skipped":0,"pct":100},"statements":{"total":1,"covered":1,"skipped":0,"pct":100},"branches":{"total":0,"covered":0,"skipped":0,"pct":100}}
,"/opt/app/lib/list.js": {"lines":{"total":26,"covered":26,"skipped":0,"pct":100},"functions":{"total":9,"covered":9,"skipped":0,"pct":100},"statements":{"total":27,"covered":27,"skipped":0,"pct":100},"branches":{"total":8,"covered":8,"skipped":0,"pct":100}}
,"/opt/app/lib/query.js": {"lines":{"total":61,"covered":61,"skipped":0,"pct":100},"functions":{"total":22,"covered":22,"skipped":0,"pct":100},"statements":{"total":65,"covered":65,"skipped":0,"pct":100},"branches":{"total":30,"covered":30,"skipped":0,"pct":100}}
,"/opt/app/lib/lang/string/snake-case.js": {"lines":{"total":3,"covered":3,"skipped":0,"pct":100},"functions":{"total":2,"covered":2,"skipped":0,"pct":100},"statements":{"total":3,"covered":3,"skipped":0,"pct":100},"branches":{"total":0,"covered":0,"skipped":0,"pct":100}}
,"/opt/app/lib/mixins/dsl.js": {"lines":{"total":111,"covered":111,"skipped":0,"pct":100},"functions":{"total":55,"covered":55,"skipped":0,"pct":100},"statements":{"total":116,"covered":116,"skipped":0,"pct":100},"branches":{"total":36,"covered":36,"skipped":0,"pct":100}}
}
```

Notice the `/opt/app/`. When I run it locally, it is `/Users/evan/dev/code/`. This normalizes that.

Reply:
Yep, that helps to see the actual output.  Thanks!